nl_server/loader.py

```diff
@@ -105,18 +107,17 @@ def _maybe_load_custom_dc_yaml():
 
 
 #
-# On prod the yaml files are in /datacommons/nl/, whereas
-# in test-like environments it is the checked in path
+# On GCP, the yaml files are in /datacommons/nl/, whereas
+# in custom DC and test-like environments it is the checked in path
 # (deploy/nl/).
 #
 def get_env_path(flask_env: str, file_name: str) -> str:
```

Comment:
@pradh @shifucun - The change we were discussing in the other thread. Please review the updated logic in here.

Reply:
I think we should keep the original logic and, for RSI, mount the folder to /datacommons/nl/.

Today, we have "climate_trace" and other instances that have NL and also need embedding file in /datacommons/nl/. We can not track every such instance in _GCP_ENVIRONMENTS

Reply:
That's a great idea. Done.

Reply:
nice!